osscluster.go

```diff
@@ -720,7 +720,7 @@ func replaceLoopbackHost(nodeAddr, originHost string) string {
 func isLoopback(host string) bool {
 	ip := net.ParseIP(host)
 	if ip == nil {
-		return true
+		return false
```

Comment:
Shoudn't `localhost` return `true` here?

Reply:
Actually, based on the network setup, many hostnames can be considered loopbacks... we should definitely handle `localhost`. `host.docker.internal` or even anything `*.docker.internal` can be considered loopback